packages/amplify-cli/src/commands/gen2-migration/codegen-generate/src/codegen-head/app_storage_definition_fetcher.ts

```diff
@@ -58,12 +61,131 @@ export class AppStorageDefinitionFetcher {
     return triggers;
   };
 
+  // Check the properties
+  private parseDynamoDBTable = async (
```

Comment:
`parse` usually means operating on a string, not fetching values from an external source.  

```suggestion
  private fetchDynamoDBTable = async (
```